publons/PublonsPlugin.inc.php

```diff
@@ -67,14 +67,15 @@ function getDescription() {
     }
 
     /**
-     * @see PKPPlugin::getTemplatePath()
+     * @see Plugin::getTemplatePath()
      */
     function getTemplatePath($inCore = false) {
-        return parent::getTemplatePath() . 'templates' . DIRECTORY_SEPARATOR;
+        $bathPath = Core::getBaseDir();
+        return 'file:' . $bathPath . DIRECTORY_SEPARATOR . parent::getTemplatePath() . DIRECTORY_SEPARATOR;
```

Comment:
this change will fix the `PublonsPlugin connection windown pop` issue, which is caused by the wrong template path
For example, if the bath path is `/var/www/html`, then the template path we want is:
`file:/var/www/html/plugins/generic/publons/templates/`
but actually what we get is:
`app:plugins/generic/publons/templatestemplates/`
which is wrong.

They remove the `file:` prefix in https://github.com/pkp/pkp-lib/commit/d38c3c867b7c70c348d83345a995ffaef557d89c#diff-6ab0758a979d98dfd8005e5cc0a077bd pull request
and the `bathPath` in https://github.com/pkp/pkp-lib/commit/fe04ade926d733a494ff8e6205f87b712738a523#diff-6ab0758a979d98dfd8005e5cc0a077bd pull request, which caused this problem